acc_test/data/valid_pivatelink_kafka_cluster_create.tf

```diff
@@ -0,0 +1,39 @@
+provider "instaclustr" {
+  username = "%s"
+  api_key = "%s"
+  api_hostname = "%s"
+}
+
+resource "instaclustr_cluster" "validPrivateLinkKafka" {
+  cluster_name = "example_kafka_privatelink"
+  node_size = "KFK-PRD-r6g.large-250"
+  data_centre = "US_WEST_2"
+  sla_tier = "NON_PRODUCTION"
+  private_network_cluster = true
+  cluster_network = "192.168.0.0/18"
+  wait_for_state = "RUNNING"
+  cluster_provider = {
+    name = "AWS_VPC"
+  }
+  rack_allocation = {
+    number_of_racks = 3
+    nodes_per_rack = 1
+  }
+
+  bundle {
+    bundle = "KAFKA"
+    version = "3.0.0"
+    options = {
+      auto_create_topics = true
+      client_encryption = false
+      delete_topics = true
+      number_partitions = 3
+      advertised_host_name = "kafka.test.com"
+      zookeeper_node_count = 3
+    }
+  }
+
+  private_link {
+    iam_principal_arns = ["arn:aws:iam::123456789012:root"]
+  }
+}
```

Comment:
**[MINOR]** New line

Reply:
Not too sure about this, is New Line at the end necessary? Because I saw other examples without New Line at the end.

Reply:
Hmm ask customerPlatform :p